rclcpp/include/rclcpp/clock.hpp

```diff
@@ -121,6 +123,16 @@ class Clock
     JumpHandler::post_callback_t post_callback,
     const rcl_jump_threshold_t & threshold);
 
+  /**
+   * Get current time from the time source specified by clock_type as nanoseconds.
```

Comment:
@BMarchi nit: `clock_type` is out of context here, can we link to `get_clock_type()` method instead?